spec/features_helper.rb

```diff
@@ -15,10 +15,13 @@
   http_client.read_timeout = 120
   http_client.open_timeout = 120
 
+  # Pin Chrome version due a possible race condition between newer versions of chromedriver and selenium
+  # causing some specs to fail intermittently on CI with the failure Net::ReadTimeout
+  # See https://github.com/teamcapybara/capybara/issues/2770
   Capybara::Selenium::Driver.new(
     app,
     browser: :chrome,
-    options: Selenium::WebDriver::Options.chrome(args:),
+    options: Selenium::WebDriver::Options.chrome(args:, browser_version: "127.0.6533.119"),
```

Comment:
discussed in person: we'd like to find a long term fix for the issues locally, thanks Leandro

Reply:
Took a different approach and changes the browser from chrome to firefox.. no need to pin a version.. tests passing locally and also on the CI.. I can amend the PR title if the new approach is accepted.